test/ansible-inventory/molecule/test-local/prepare.yml

```diff
@@ -25,12 +25,9 @@
       delegate_to: localhost
 
     - name: Wait for the Kubernetes API to become available (this could take a minute)
-      uri:
-        url: "https://localhost:8443/version"
-        status_code: 200
-        validate_certs: no
+      command: kubectl cluster-info
       register: result
-      until: (result.status|default(-1)) == 200
+      until: result.stdout is regex("Kubernetes master.* is running")
```

Comment:
Note that regex is required because the actual output is colorized, meaning it comes to Ansible as:

```
\u001b[0;32mKubernetes master\u001b[0m is running at \u001b[0;33mhttps://localhost:8443\u001b[0m
```